src/ascend.ts

```diff
@@ -21,7 +20,29 @@ export enum Lifestyle {
   hardcore = 3,
 }
 
-function toMoonId(moon: string | number, playerClass: Class): number {
+type MoonSign =
```

Comment:
I almost want to break this into like 10 types and make it a union of those types, even though there's no utility to that.

Reply:
at some point we should have a moon sign type either in mafia or in libram that can be used anywhere.